src/components/details/details.js

```diff
@@ -76,76 +78,65 @@ Details.prototype.getAncestor = function (node, match) {
   return node
 }
 
-/**
-* Initialise the script on a list of details elements in a container
-* @param {object} list of details elements
-* @param {string} container where to look for details elements
-*/
-Details.prototype.init = function (list, container) {
-  container = container || document.body
+Details.prototype.init = function () {
+  var $module = this.$module
 
-  // Get the collection of details elements, but if that's empty
-  // then we don't need to bother with the rest of the scripting
-  if ((list = container.getElementsByTagName('details')).length === 0) {
+  if (!$module) {
     return
   }
-  // else iterate through them to apply their initial state
-  var n = list.length
-  var i = 0
-  for (i; i < n; i++) {
-    var details = list[i]
-
-    // Save shortcuts to the inner summary and content elements
-    details.__summary = details.getElementsByTagName('summary').item(0)
-    details.__content = details.getElementsByTagName('div').item(0)
-
-    // If <details> doesn't have a <summary> and a <div> representing the content
-    // it means the required HTML structure is not met so the script will stop
-    if (!details.__summary || !details.__content) {
-      return
-    }
 
-    // If the content doesn't have an ID, assign it one now
-    // which we'll need for the summary's aria-controls assignment
-    if (!details.__content.id) {
-      details.__content.id = 'details-content-' + i
-    }
+  var details = $module
 
-    // Add ARIA role="group" to details
-    details.setAttribute('role', 'group')
+  // Save shortcuts to the inner summary and content elements
+  $module.__summary = $module.getElementsByTagName('summary').item(0)
```

Comment:
Not a blocker and I know they were already there, but we're using double underscores only on this component when storing references.

Reply:
Forget about that, you already changed it in the next commit.